shared/docker-entrypoint.sh

```diff
@@ -30,6 +30,8 @@ elif [ -n "$S3_URI" ]; then
   if [ -z "$S3_SECRET_ACCESS_KEY" ]; then
     opts="--no-sign-request"
   fi
+  # Setting Signature Version 4 for S3 Request Authentication
+  aws configure set s3.signature_version s3v4
```

Comment:
Should this be `aws configure set default.s3.signature_version s3v4`? Took this from https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingAWSSDK.html

Reply:
ah nvm i'm thinking it's really just used when you're configuring the cli on a local machine maybe

Reply:
Yeah if you don't specify it adds it to default, here's what's been added to the `~/.aws/config` after running the command on my machine:
```
[default]
s3 =
    signature_version = s3v4
```